src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,39 @@
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.store.basic.UserStore" %>
+<%
+List<Message> messages = (List<Message>) request.getAttribute("messages");
```

Comment:
Do we need this line?

Reply:
I added this for later use because we're going to be iterating through the messages sent by the user whose profile we're viewing but you're right, this isn't needed for the prototype.

Reply:
Cool, it's fine to keep this one. In general, it's good to keep the code scoped to the change though. If we need to revert a change for any reason in the future, it should only revert the functionality it added, nothing more or less. :)